packages/index.yaml

```diff
@@ -48,3 +48,7 @@ packages:
     latestVersion: v2.15.3+1
     name: sealed-secrets
     shortDescription: A Kubernetes controller and tool for one-way encrypted Secrets
+  - iconUrl: "https://avatars.githubusercontent.com/u/36015203"
```

Comment:
Please remove the changes to this file, as it will be updated automatically by our pipeline.